Framework/DataHandling/test/SetSampleTest.h

```diff
@@ -223,6 +255,25 @@ class SetSampleTest : public CxxTest::TestSuite {
         1e-08);
   }
 
+  void test_Setting_Environment_Without_Sample() {
+    using Mantid::Geometry::SampleEnvironment;
+    using Mantid::Kernel::ConfigService;
+
+    auto inputWS = WorkspaceCreationHelper::create2DWorkspaceBinned(1, 1);
+    auto testInst = ComponentCreationHelper::createTestInstrumentCylindrical(1);
+    testInst->setName(m_instName);
+    inputWS->setInstrument(testInst);
+
+    // Algorithm uses instrument directories as a search location, alter this
+    // for the test
+    auto &config = ConfigService::Instance();
+    const auto defaultDirs = config.getString("instrumentDefinition.directory");
+    config.setString("instrumentDefinition.directory", m_testRoot);
+    auto alg = createAlgorithm(inputWS);
+    alg->setProperty("Environment", createEnvironmentProps("10mm_empty"));
+    TS_ASSERT_THROWS_NOTHING(alg->execute());
+  }
```

Comment:
As we're changing the config here I think it would be good to reset it to the defaults once the test has finished so it is in the correct state for other tests. I think adding `config.setString("instrumentDefinition.directory", defaultDirs )` at the end should work.

Reply:
Similarly for the other tests added in this file